### DISCUSSION TITLE
Even though I am approved , my github education benefits are still not available

### DISCUSSION THREAD

**Discussion by aglizb** [community/community#142749] [GitHub Education]
### Hi everyone,

Hi everyone,
I recently got approved for Github Education but I have not gotten my access to the benefits and I am worried because it has been a few days. Is it possible to know when I can get it . Thank you !

**Comment by davevad93** [comment]
Hi @aglizb ,

Don't worry, the waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @aglizb was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Comment by thomassiard** [comment]
It can sometimes take a few days for the benefits to activate, but if it’s been more than a week, it might be worth double-checking a few things:

Check Your Email: Sometimes, benefits activation details are sent via email, and these emails can end up in your spam or promotions folder by mistake.

Log In to GitHub Education: Try logging in at [GitHub Education](https://education.github.com/) to see if your benefits are active on your account. You may need to refresh or log out and log back in to see the updates.

Contact GitHub Support: If it’s still not showing up, reaching out to GitHub Support can help them look into your account and speed up the process.
